libs/labelbox/src/labelbox/schema/dataset.py

```diff
@@ -264,13 +257,16 @@ def create_data_rows_sync(self, items) -> None:
             url_param: descriptor_url
         })
 
-    def create_data_rows(self, items) -> "Task":
+    def create_data_rows(self,
+                         items,
+                         file_upload_thread_count=FILE_UPLOAD_THREAD_COUNT
```

Comment:
We should validate that thread_count is a positive integer here and raise an error if not